DiamondLightSource/SynchWeb
# LIMS-1204: Remove log area from top of active visits

**JIRA ticket**: [LIMS-1204](https://jira.diamond.ac.uk/browse/LIMS-1204)

**Summary**:

At the top of active visits, there is a 4 line 'log' area, which updates to show new data collections. This adds no extra functionality as the page already updates, and on page 2 onwards it doesnt work correctly. So this is to remove it.

**Changes**:
- Remove the <div class="lg"> section, from each data collection view
- Remove the javascript that populated it
- Add a nopadding class to prevent too much white space
- Edit the webcam/status link to say 'Show' or 'Hide' and to have a down or up arrow as appropriate
- Remove the outdated reference to i02, which had 3 webcams displayed, and the style associated with it

**To test**:
- Go to an active visit in Synchweb, ensure the other elements are still present:
  - "Data Collections for..." heading
  - "Assign Containers", button "Summary" button etc
  - Show (beamline) Webcams and Beamline Status link, with downward arrow
  - Search bar
  - Filters for "Data Collections", "Grid Scans" etc
  - Page numbers and number per page
- Expand the webcams and check the heading switches from "Show" to "Hide" and the arrows turn upwards.
- Go to inactive visit and check everything looks as before, with the same elements as above but without the Webcams/Beamline Status section
